test_goto.py

```diff
@@ -63,6 +63,18 @@ def func():
     assert func() == 0
 
 
+def test_jump_out_of_loop_and_survive():
```

Comment:
I might (likely) miss something here, but how is this new test related to adding support for Python 3.8?